src/OrphanFileHelper.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+
+const logger = require('./logging');
+const fileStore = require('./store/file');
+
+/**
+ *
+ */
+class OrphanFileHelper {
+    /**
+     *
+     */
+    constructor({ tempPath, reconnectTimeout, dumpPersister }) {
+        this.tempPath = tempPath;
+        this.reconnectTimeout = reconnectTimeout;
+        this.dumpPersister = dumpPersister;
+    }
+
+    /**
+     * Remove old files from the temp folder.
+     */
+    processOldFiles() {
+        logger.info('[OrphanFileHelper] Waiting for connections to reconnect.');
+        try {
+            if (fs.existsSync(this.tempPath)) {
+                fs.readdirSync(this.tempPath).forEach(fname => {
+                    try {
+                        const filePath = `${this.tempPath}/${fname}`;
+
+                        logger.debug(`[OrphanFileHelper] Trying to process file ${filePath}`);
+                        fs.stat(filePath, (err, stats) => {
+                            if (err) {
+                                throw err;
+                            }
+
+                            this.processIfExpired(stats, filePath, fname);
+                        });
+                    } catch (e) {
+                        logger.error(`[OrphanFileHelper] Error while unlinking file ${fname} - ${e}`);
+                    }
+                });
+            } else {
+                logger.error('[OrphanFileHelper] Temp path doesn\'t exists. path: ', this.tempPath);
```

Comment:
If temp path doesn't exists, we might want to kill everything as this is unexpected, consider letting the exception bubble up.